rails.civmap.json

```diff
@@ -396,3 +396,4 @@
 {"name":"Pripyat - Appomattox","type":"rail_edge","id":"1805d0b5-e3cc-42d2-937b-19ad9108b470","line":[[[6194,2330],[6999,2330]]],"accuracy":3,"elevation":"underground","progress":"finished","two_way":true}],"presentations":[{"name":"Rails and Stops","style_highlight":{"stroke_color":"#ff0000"},"style_base":{"stroke_color":{"feature_key":"elevation","default":"#808080","categories":{"underground":"#00ff00","surface":"#00ffff"}},"dash_array":{"feature_key":"progress","default":null,"categories":{"planned":[0,10],"started":[5,10]}},"stroke_width":{"feature_key":"type","default":2,"categories":{"rail_edge":5}},"color":"#ffffff","icon":"circle","icon_size":{"feature_key":"stop_importance","default":16,"categories":{"minor":12}}}},
 {"name":"Rails only","style_base":{"opacity":{"feature_key":"type","default":1,"categories":{"rail_node":0}},"stroke_color":{"feature_key":"elevation","default":"#808080","categories":{"underground":"#00ff00","surface":"#00ffff"}},"dash_array":{"feature_key":"progress","default":null,"categories":{"planned":[0,10],"started":[5,10]}},"stroke_width":{"feature_key":"type","default":2,"categories":{"rail_edge":5}},"color":"#ffffff","icon":"circle","icon_size":{"feature_key":"stop_importance","default":16,"categories":{"minor":12}}}},
 {"name":"Stops only","style_base":{"opacity":{"feature_key":"type","default":1,"categories":{"rail_edge":0}},"stroke_color":{"feature_key":"elevation","default":"#808080","categories":{"underground":"#00ff00","surface":"#00ffff"}},"dash_array":{"feature_key":"progress","default":null,"categories":{"planned":[0,10],"started":[5,10]}},"stroke_width":{"feature_key":"type","default":2,"categories":{"rail_edge":5}},"color":"#ffffff","icon":"circle","icon_size":{"feature_key":"stop_importance","default":16,"categories":{"minor":12}}}}],"enabled_presentation":"Rails and Stops","id":"ccmap/rail/collection","info":{"version":"3.0.0-beta3","last_update":1555440266},"name":"Rails","source":"data/rails.civmap.json"}
+{"name":"Iria Savion","type":"rail_edge","id":"b852b863-12ef-4f71-a5d7-c4296de49742","line":[[[10309,-778],[10500,-778],[10500,-6000],[10047,-6000],[10047,-6493]]],"accuracy":3,"progress":"finished","elevation":"underground","two_way":true,"ice":false},
```

Comment:
this is invalid json. move your new line between some other lines, then use https://jsonlint.com/ or something similar to check for errors